stories/skeleton.stories.js

```diff
@@ -37,3 +37,21 @@ export const Stat = () => (
     <Skeleton view='stat'/>
   </div>
 )
+
+export const Line = () => (
+  <div style={{ width: '500px' }}>
+    <Skeleton view='line'/>
```

Comment:
Let's define a SKELETON_TYPE constant and use it instead of all strings in the code: 'line' -> SKELETON_TYPE.line
Add to it whatever is missing.
```
const SKELETON_TYPE = [
  'line',
  'bar',
  'barline',
  'pie',
  'map',
  'stat',
  'pyramid',
].reduce((a, v) => ({ ...a, [v]: v }), {})
```

Reply:
Oh, I think I forgot to mention the STAT widget skeleton. Sorry about that. Could you please add it when Veronica has a design in place? You can do that in a next PR.

Reply:
> Oh, I think I forgot to mention the STAT widget skeleton. Sorry about that. Could you please add it when Veronica has a design in place? You can do that in a next PR.

Sorry, I was wrong about this ^: @vlatawiec @Hamaad102.